app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
+              text: "History of government",
+            }
+          ]
+        } %>
       </div>
 
-      <div class="full-width-section ministers">
+      <section>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Who runs government",
+          id: "who-runs-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+        
+        <section class="govuk-!-margin-bottom-9 prime-minister">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: "The Prime Minister",
+            font_size: "m",
+            heading_level: 3,
+            margin_bottom: 4,
+          } %>
+
+          <p class="govuk-body">The <%= link_to("Prime Minister", "/government/ministers/prime-minister", class: "govuk-link") %> is the leader of Her Majesty’s Government and is ultimately responsible for all policy and decisions.</p>
+          <p class="govuk-body">The Prime Minister also:</p>
+
+          <%= render "govuk_publishing_components/components/list", {
+            visible_counters: true,
+            items: [
+              "oversees the operation of the Civil Service and government agencies",
+              "appoints members of the government",
+              "is the principal government figure in the House of Commons",
+            ]
+          } %>
+
+          <% if @prime_minister.present? %>
+            <p class="govuk-body">The Prime Minister is <%= link_to @prime_minister.name, url_for(@prime_minister), class: "govuk-link" %>.</p>
+          <% end %>
+
+          <p class="govuk-body govuk-!-margin-bottom-8">
+            <%= link_to "Read more about the Prime Minister's Office, 10 Downing Street", "/government/organisations/prime-ministers-office-10-downing-street", class: "govuk-link" %>
+          </p>
+          
+          <% if @prime_minister.present? && @prime_minister.image_url(:s465).present? %>
+            <%= image_tag @prime_minister.image_url(:s465), { alt: @prime_minister.name } %>
+          <% else %>
+            <%= image_tag 'how-gov-works/10_Downing_Street.jpg', alt: 'Number 10, Downing Street' %>
+          <% end %>
+        </section>
+
+        <section class="govuk-!-margin-bottom-9">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: "The Cabinet",
+            font_size: "m",
+            heading_level: 3,
+            margin_bottom: 4,
+          } %>
+
+          <p class="govuk-body">The Cabinet is made up of the senior members of government. Every week during Parliament, members of the Cabinet (Secretaries of State from all departments and some other ministers) meet to discuss the most important issues for the government.</p>
+          <p class="govuk-body govuk-!-margin-bottom-8"><%= link_to("See who is in the Cabinet", "/government/ministers", class: "govuk-link") %></p>
+          
+          <%= image_tag 'how-gov-works/Cabinet_01.jpg', alt: 'The Cabinet' %>
```

Comment:
I don't think this Content is apt, something like "The Cabinet Room at No. 10 Downing Street Cabinet table" I think is more descriptive of what this image is.

Reply:
Provided empty) alt text for the following images -
```
<%= image_tag @prime_minister.image_url(:s465), { alt: '' } %>
<%= image_tag 'how-gov-works/10_Downing_Street.jpg', alt: '' %>
<%= image_tag 'how-gov-works/Cabinet_01.jpg', alt: '' %>
<%= image_tag 'how-gov-works/Churchill_01.jpg', alt: '' %>
```